src/sys/unix/uds/listener.rs

```diff
@@ -80,7 +80,13 @@ pub(crate) fn accept(listener: &net::UnixListener) -> io::Result<(UnixStream, So
         // Ensure the socket is closed if either of the `fcntl` calls
         // error below.
         let s = unsafe { net::UnixStream::from_raw_fd(socket) };
-        syscall!(fcntl(socket, libc::F_SETFD, libc::FD_CLOEXEC)).map(|_| s)
+        syscall!(fcntl(socket, libc::F_SETFL, libc::O_NONBLOCK))
```

Comment:
Can you also do this in `src/sys/unix/tcp.rs`, after `stream` is created on line 469. Also limited to x86 Android.

https://github.com/tokio-rs/mio/blob/527e843b0e11d585c0b20a8a3e4df966766f8bfa/src/sys/unix/tcp.rs#L461-L469